parts/params.t

```diff
@@ -6,6 +6,7 @@
     },
     "vmSize": {
       "type": "string",
+      {{GetDefaultVMSize}}
```

Comment:
I would suggest following {{GetOsDiskTypes}} example and using getAllowedDefaultValues()